docs/integrations/data-ingestion/azure-data-factory/index.md

```diff
@@ -0,0 +1,481 @@
+import adfNewLinkedServiceParams                from '@site/static/images/integrations/data-ingestion/azure-data-factory/adf-new-linked-service-params.png';
+import adfNewLinkedServiceExpressionFieldFilled from '@site/static/images/integrations/data-ingestion/azure-data-factory/adf-new-linked-service-expression-field-filled.png';
+import adfNewLinkedServiceCheckConnection       from '@site/static/images/integrations/data-ingestion/azure-data-factory/adf-new-linked-service-check-connection.png';
+import adfLinkedServicesList                    from '@site/static/images/integrations/data-ingestion/azure-data-factory/adf-linked-services-list.png';
+import adfNewDatasetItem                        from '@site/static/images/integrations/data-ingestion/azure-data-factory/adf-new-dataset-item.png';
+import adfNewDatasetPage                        from '@site/static/images/integrations/data-ingestion/azure-data-factory/adf-new-dataset-page.png';
+import adfNewDatasetProperties                  from '@site/static/images/integrations/data-ingestion/azure-data-factory/adf-new-dataset-properties.png';
+import adfNewDatasetQuery                       from '@site/static/images/integrations/data-ingestion/azure-data-factory/adf-new-dataset-query.png';
+import adfNewDatasetConnectionSuccessful        from '@site/static/images/integrations/data-ingestion/azure-data-factory/adf-new-dataset-connection-successful.png';
+import adfNewPipelineItem                       from '@site/static/images/integrations/data-ingestion/azure-data-factory/adf-new-pipeline-item.png';
+import adfNewCopyDataItem                       from '@site/static/images/integrations/data-ingestion/azure-data-factory/adf-new-copy-data-item.png';
+import adfCopyDataSource                        from '@site/static/images/integrations/data-ingestion/azure-data-factory/adf-copy-data-source.png';
+import adfCopyDataSinkSelectPost                from '@site/static/images/integrations/data-ingestion/azure-data-factory/adf-copy-data-sink-select-post.png';
+import adfCopyDataDebugSuccess                  from '@site/static/images/integrations/data-ingestion/azure-data-factory/adf-copy-data-debug-success.png';
+
+# Bringing Azure Data into ClickHouse
+
+Microsoft Azure offers a wide range of tools to store, transform, and analyze
+data. However, in many scenarios, ClickHouse can provide significantly better
+performance for low-latency querying and processing of very large datasets. In
+addition, ClickHouse's columnar storage and compression can greatly reduce the
+cost of querying large volumes of analytical data compared to general-purpose
+Azure databases.
+
+In this article, we will explore two ways to ingest data from Microsoft Azure
+into ClickHouse. The first method, and likely the easiest, involves using
+ClickHouse’s `azureBlobStorage` table function to transfer data directly from
```

Comment:
I wonder if `azure blob storage` integration should be a separate page. From the Azure Data Factory point of view, Azure Blob Storage is just a well-integrated data source and sink. WDYT? @laeg